e2e_playwright/st_components_v1_test.py

```diff
@@ -28,7 +28,10 @@ def test_components_iframe_rendering(
     expect(elements).to_have_count(3)
 
     assert_snapshot(elements.nth(0), name="st_components-html")
-    assert_snapshot(elements.nth(1), name="st_components-iframe")
+    # The second iframe does not use a valid URL, and therefore will show
```

Comment:
Is it possible to just remove this?

Reply:
I removed it but added a short comment above the other snapshot